packages/cadl-ranch-specs/http/azure/core/lro/rpc/main.tsp

```diff
@@ -95,4 +95,10 @@ Expected response body:
 ```
 """)
 @route("/generations:submit")
-op longRunningRpc is global.Azure.Core.LongRunningRpcOperation<GenerationOptions, GenerationResponse, GenerationResult>;
+op longRunningRpc is global.Azure.Core.LongRunningRpcOperation<
+  {
+    @body body: GenerationOptions;
```

Comment:
Any ask on the naming of the request body property?

As `body`, there will be a name change in Java (but it does not break API)
```diff
-    public PollerFlux<PollOperationDetails, GenerationResult> beginLongRunningRpc(GenerationOptions generationOptions) {
+    public PollerFlux<PollOperationDetails, GenerationResult> beginLongRunningRpc(GenerationOptions body) {
```

Reply:
Here the model is all body properties, so `@body` and `@bodyRoot` should be equivalent.

Reply:
in tcgc logic, if the body model has name, then use that name, else use `body`.

Reply:
Note that Java prefer e.g. `requestContext` instead of `body`.